src/components/GraphiQL.js

```diff
@@ -322,6 +323,7 @@ export class GraphiQL extends React.Component {
                 value={this.state.query}
                 onEdit={this._onEditQuery}
                 onHintInformationRender={this._onHintInformationRender}
+                editorOptions={this._getEditorOptions()}
```

Comment:
This should be `this.props.editorOptions` based on the added prop type, correct?